### PR TITLE
Issue #8105 - Update PossibleBeaconingActivity.yaml

### PR DESCRIPTION
**Pull request by azurekid**
Added option for private IP ranges per request #8105 Alignment of code for readability

   Required items, please complete
   
   Change(s):
   - Updated file PossibleBeaconingActivity.yaml

   Reason for Change(s):
   - #8105 

   Version Updated:
   - [x] version updated to 1.1.3

   Testing Completed:
   - [x] Tested in own environment

### CODE REVIEW

**Comment by v-rbajaj** [comment]
Thanks @azurekid for raising the PR, we will soon update you.

**Comment by vakohl** [comment]
ETA for this review is 18th Aug

File: Solutions/Network Session Essentials/Analytic Rules/PossibleBeaconingActivity.yaml [hunk 1]
```diff
@@ -77,21 +77,60 @@ query: |
   let TimeDeltaThreshold = 10;
   let TotalEventsThreshold = 15;
   let PercentBeaconThreshold = 80;
+  let LocalNetworks=dynamic(["169.254.0.0/16","127.0.0.0/8"]);
   _Im_NetworkSession(starttime=ago(querystarttime), endtime=ago(queryendtime))
   | where not(ipv4_is_private(DstIpAddr))
-  | project TimeGenerated,   SrcIpAddr, SrcPortNumber, DstIpAddr, DstPortNumber, DstBytes, SrcBytes
-  | sort by SrcIpAddr asc,TimeGenerated asc, DstIpAddr asc, DstPortNumber asc
+  | where not (ipv4_is_in_any_range(DstIpAddr, LocalNetworks))
```
Comment (vakohl) [inline Solutions/Network Session Essentials/Analytic Rules/PossibleBeaconingActivity.yaml:83]: @azurekid can we use - not (ipv4_is_private(DstIpAddr)) for taking only internet traffic ?
https://learn.microsoft.com/en-us/azure/data-explorer/kusto/query/ipv4-is-privatefunction

Comment (vakohl) [reply Solutions/Network Session Essentials/Analytic Rules/PossibleBeaconingActivity.yaml:83]: @azurekid I just noticed that filter is being used already at line 82. Your change looks good, please ignore my previous comment

**Review by vakohl — APPROVED** [review]

File: Solutions/Network Session Essentials/Analytic Rules/PossibleBeaconingActivity.yaml (hunk 1 above)
Comment (azurekid) [reply Solutions/Network Session Essentials/Analytic Rules/PossibleBeaconingActivity.yaml:83]: No worries, according to the documentation some ranges are nor excluded via the private-id function.

Therefore the let statement is used. 👍

**Review by v-atulyadav — APPROVED** [review]

**Pull request merged**